bungiesearch/utils.py

```diff
@@ -40,20 +41,27 @@ def update_index(model_items, model_name, bulk_size=100, num_docs=-1, start_date
                     if end_date:
                         model_items = model_items.filter(**{'{}__lte'.format(index_instance.updated_field): __str_to_tzdate__(end_date)})
 
-                logging.info('Fetching number of documents to be added to {}.'.format(model.__name__))
+                logging.info('Fetching number of documents to update in {}.'.format(model.__name__))
```

Comment:
Could this be:
`logging.info('Fetching number of documents to {} in {}.'.format(action, model.__name__))`, just like on line 47 ;)